src/components/overview/cart.js

```diff
@@ -1,26 +1,23 @@
-import React, { useState, useEffect } from "react";
-import { useProduct } from "../../contexts/ProductContext.js";
-import axios from "axios";
-import InputLabel from '@mui/material/InputLabel';
-import MenuItem from '@mui/material/MenuItem';
-import FormControl from '@mui/material/FormControl';
-import Select from '@mui/material/Select';
```

Comment:
Material UI dependencies are removed. Why was this done, and does this have any other effects on the project.